server/users.ts

```diff
@@ -533,11 +535,11 @@ export class User extends Chat.MessageContext {
 		const auth = (room && !this.can('makeroom') ? room.auth.get(this.id) : this.group);
 		return auth in Config.groups && Config.groups[auth].rank >= Config.groups[minAuth].rank;
 	}
-	can(permission: RoomPermission, target: User | null, room: BasicRoom): boolean;
+	can(permission: RoomPermission, target: User | null, room: Room | BasicChatRoom, cmd?: string): boolean;
```

Comment:
```suggestion
	can(permission: RoomPermission, target: User | null, room: Room | BasicRoom, cmd?: string): boolean;
```

I'm surprised why this isn't an error, because `BasicChatRoom` doesn't exist anymore.